classes/i18n/interfaces/LocaleInterface.php

```diff
@@ -73,11 +72,6 @@ public function registerLoader(callable $fileLoader, int $priority = 0): void;
      */
     public function isLocaleValid(?string $locale): bool;
 
-    /**
-     * Check if the supplied submission locale is valid.
-     */
-    public function isSubmissionLocaleValid(?string $locale): bool;
```

Comment:
If this was introduced before 3.5, then better to place a `@deprecated`

Reply:
First introduced for 3.5